src/views/FindPark.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <section>
+    <input
+      class="search-input"
+      type="text"
+      placeholder="Search by name, city or zip code"
+      v-model="searchTerm"
+    />
+    <button class="search-button" @click="search('searchTerm')">
+      Get Started - woof!
+    </button>
+    <p v-if="searchTerm === ''">Please enter a park name, city, or zip code</p>
```

Comment:
I like the placeholder and render for instructions! Great for user empathy that may have some troubles seeing the placeholder text inside of the input.

Reply:
I agree with Nathan.  Great job!  We should add some styling for the `<p>` tag's error message so that it stands out.  We could also look into the best way to highlight the border of the input field as a secondary measure for notification of the error.  I'll add an issue for the project board for that, if you agree.